### ISSUE TITLE
Program not working under Windows

### ISSUE THREAD

**Issue by Hoffenbar** [FauconFan/mdbook-cmdrun#5]
I've tested several variations, e.g.
```
<!-- cmdrun tclkitsh test1.tcl -->

<!-- cmdrun cmd /cecho abcd -->
```
but no output is produced. Maybe because you call sh instead of env(COMSPEC)?

**Comment by FauconFan** [comment]
Hi!
I am not familiar enough with the Windows OS shells, so I don't know how to fix this... (I also don't have the time).
I mark this as Help Wanted and let you (or anyone who feel confident) to solve this.
It would be nice to have to replicate the test pipeline in Windows as well. (If you (or someone) wants to hotfix this without replicating the pipeline, I'll open another issue)

**Comment by Hoffenbar** [comment]
Unfortunally, I'm not a rust-guy, so I could not solve this by myself..... Thank you for your response!

**Comment by kevinglasson** [comment]
I think this would help:

```text
https://doc.rust-lang.org/std/process/struct.Command.html
```

Particularly

```rust
use std::process::Command;

let output = if cfg!(target_os = "windows") {
    Command::new("cmd")
            .args(["/C", "echo hello"])
            .output()
            .expect("failed to execute process")
} else {
    Command::new("sh")
            .arg("-c")
            .arg("echo hello")
            .output()
            .expect("failed to execute process")
};

let hello = output.stdout;
```

**Comment by FauconFan** [comment]
This has been resolved by @exsjabe in #10. Thanks to him!
